Exiled.API/Structs/AttachmentIdentifier.cs

```diff
@@ -138,14 +140,14 @@ internal AttachmentIdentifier(uint code, AttachmentName name, AttachmentSlot slo
         /// <returns><see langword="true"/> if <see cref="string"/> was converted successfully; otherwise, <see langword="false"/>.</returns>
         public static bool TryParse(string s, out AttachmentIdentifier identifier)
         {
-            identifier = default;
-
-            foreach (AttachmentIdentifier attId in Features.Items.Firearm.AvailableAttachments.Values.SelectMany(kvp => kvp.Where(kvp2 => kvp2.Name.ToString() == s)))
+            foreach(var attIds in Features.Items.Firearm.AvailableAttachments.Values)
```

Comment:
Don't really get it.
As long as it works and it has been tested, I guess it's fine.
I need your feedback though.

Reply:
I would make these changes in another push and explain them more clearly.